shoes-core/lib/shoes/logger.rb

```diff
@@ -1,66 +1,47 @@
 # frozen_string_literal: true
+require 'logger'
+
 class Shoes
-  module Logger
-    class << self
-      def register(name, obj)
-        @loggers ||= {}
-        @loggers[name] = obj
-      end
+  class Logger
+    def initialize
+      @loggers = []
+    end
 
-      def unregister(name)
-        @loggers.delete(name)
-      end
+    def <<(logger)
+      @loggers << logger
+      self
+    end
```

Comment:
Agreed that the naming didn't end up quite as clear as I'd have liked on first pass, and I totally see what you mean. I see two possible renames that might address it:

* My fancy OO patterns would have me call `Shoes::Logger` a `CompositeLogger` which makes it clearer that it's purpose is to dispatch to other loggers.
* Alternatively, we could also just treat `Shoes::Logger` as a `LoggerCollection` instead. `<<` is more natural in that context, although I'm not opposed to just renaming to `add_logger` if you find that clearer.

What do you think? Either of those work for you @PragTob?